src/LightningInspectorInjectedScript.js

```diff
@@ -1072,7 +1072,9 @@ function addActionRules() {
         var removeClassName = 'auraDevToolServiceHighlight3';
         var addClassName = 'auraDevToolServiceHighlight4';
         var element = event.target;
-        element.classList.remove(removeClassName);
-        element.classList.remove(addClassName);
+        if (element) {
+            element.classList.remove(removeClassName);
```

Comment:
names are a bit weird for them both to be removed

Reply:
They show the highlights when you hover over them in the component tree.

If you don't remove both, then the page becomes unusable.